File: C#/coding challenges/oopiracy.cs
// https://www.codewars.com/kata/54fe05c4762e2e3047000add/train/csharp

public class Ship
{
  public int Draft;
  public int Crew;
  
  public Ship(int draft, int crew)
  {
    Draft = draft;
    Crew = crew;
  }
  
  public bool IsWorthIt()
  {
    double weight = this.Draft - (this.Crew * 1.5);
    if (weight >= 20){
    return true;
    } else {
    return false;
    }
    
  }
  
}

File: python/coding challenges/oopiracy.py
# https://www.codewars.com/kata/54fe05c4762e2e3047000add/train/python

class Ship:
    def __init__(self, draft, crew):
        self.draft = draft
        self.crew = crew
    def is_worth_it(cheese):
        weight = cheese.draft - (cheese.crew * 1.5)
        if (weight >= 20):
            return True
        else:
            return False